backend/views/syson-diagram-common-view/src/main/java/org/eclipse/syson/diagram/common/view/services/ViewCreateService.java

```diff
@@ -565,17 +571,83 @@ public boolean isEmptyAcceptActionUsagePayload(Element element) {
+     * Add the standard start action as the child of the given element.
+     *
+     * @param ownerElement
+     *            an element that will own the standard start action.
+     * @return the {@link Membership} element containing the start action in its memberElement feature.
+     */
+    public Membership addStartAction(Element ownerElement) {
+        var standardStartAction = this.utilService.retrieveStandardStartAction(ownerElement);
+        if (standardStartAction != null) {
+            var membership = SysmlFactory.eINSTANCE.createMembership();
+            membership.setMemberElement(standardStartAction);
+            ownerElement.getOwnedRelationship().add(membership);
+            return membership;
+        }
+        return null;
+    }
 
-        Succession succession = SysmlFactory.eINSTANCE.createSuccession();
-        this.elementInitializerSwitch.doSwitch(succession);
+    /**
+     * Create a new action {@link ActionUsage} inside the given element which should be an {@link ActionUsage} or an
+     * {@link ActionDefintion}.
+     *
+     * @param ownerElement
+     *            the owner of the new action usage.
+     * @return the newly created action usage.
+     */
+    public ActionUsage createSubActionUsage(Element ownerElement) {
+        var newActionUsage = SysmlFactory.eINSTANCE.createActionUsage();
+        this.elementInitializerSwitch.doSwitch(newActionUsage);
         var featureMembership = SysmlFactory.eINSTANCE.createFeatureMembership();
-        featureMembership.getOwnedRelatedElement().add(succession);
-        succession.getSource().add(sourceAction);
-        succession.getTarget().add(targetAction);
-        sourceParentElement.getOwnedRelationship().add(featureMembership);
+        featureMembership.getOwnedRelatedElement().add(newActionUsage);
+        ownerElement.getOwnedRelationship().add(featureMembership);
+        return newActionUsage;
+    }
 
-        return sourceAction;
+    /**
+     * Removal service for Start action inside an action usage or definition.
+     *
+     * @param selectedNode
+     *            the node element that represents the start action in the diagram.
+     * @param editingContext
+     * @param diagramService
+     * @return the element that owned the start action.
+     */
+    public Element removeStartAction(Node selectedNode, IEditingContext editingContext, IDiagramService diagramService) {
+        Element owner = this.getSourceOwner(selectedNode, editingContext, diagramService);
+        var membership = owner.getOwnedRelationship().stream()
+                .filter(Membership.class::isInstance)
+                .map(Membership.class::cast)
+                .filter(m -> {
+                    return m.getMemberElement() instanceof ActionUsage au && this.utilService.isStandardStartAction(au);
+                })
+                .findFirst()
+                .orElse(null);
+        if (membership != null) {
+            EcoreUtil.remove(membership);
```

Comment:
Please prefer the use of `DeleteService.deleteFromModel`

Reply:
Done